concepticondata/conceptlists.tsv

```diff
@@ -383,6 +383,7 @@ Teit-1908-693	Teit, James	1908		693	questionnaire, areal	English	Okagan language
-Goddard-2002-59	Goddard, Cliff and Wierzbicka, Anna	2002		59	specific	English	Global		Goddard2002		This list contains concept labels for 59 semantic primes sorted into different categoried based on [Natural Semantic Metalanguage](https://de.wikipedia.org/wiki/Natural_Semantic_Metalanguage)) approach. The authors claim the concepts to be universal.	14	
+Goddard-2002-59	Goddard, Cliff and Wierzbicka, Anna	2002		59	specific	English	Global		Goddard2002		This list contains concept labels for 59 semantic primes sorted into different categories based on [Natural Semantic Metalanguage](https://de.wikipedia.org/wiki/Natural_Semantic_Metalanguage)) approach. The authors claim the concepts to be universal.	14	
 Goz-2017-600	Göz, İlyas and Tekcan, Ali İ. and Erciyes, Aslı Aktan	2017		600	ratings	English, Turkish	Turkish		Goz2017		The list contains ratings on age-of-acquisition for Turkish words (nouns and adjectives) across different age groups. In addition, ratings for imageability and concreteness from [Tekcan and Göz (2005)](:bib:Tekcan2005) as well as word frequency from [Göz (2003)](:bib:Goz2003) are included.		
 Feleke-2021-150	Feleke, Tekabe Legesse	2021		150	specific, areal	English	Ethiosemitic languages	https://www.sciencedirect.com/science/article/pii/S2215039021000023	Feleke2021		The list was used in a study for the areal classification of Ethiosemitic languages and to establish distances/relationships between the members of this group. According to the author, the distances were calculated by comparing cognates from randomly selected 147 words. Note that the article mentions that the list contains 147 items, whereas the actual supplementary materials consist of 150 items.		
+Nurse-1975-1038	Nurse, Derek and Philippson, Gérard	1975		1038	basic, areal	English	Bantu languages		Nurse1975		…		
```

Comment:
the note is best based on the Website description: http://www.cbold.ish-lyon.cnrs.fr/Docs/Swa-Eng1.html and you should add the link to the concept list online: http://www.cbold.ish-lyon.cnrs.fr/Docs/Swa-Eng1.html

> Tanzania Language Survey was conducted in the early 1970s, but never published as a hole. As a result, the data we have for it is based on the description in the [Comparative Bantu Online Dictionary](http://www.cbold.ish-lyon.cnrs.fr/Docs/TLSinfo.html), which provides a partially digitized version of the original data and also the source of the concept list we provide here. Since the original concept list, being part of the database, faced apparent limits due to computational machinery (glosses of a certain length were exceeding the limit of a cell), there are varying numbers of concepts that have been discussed. The website talks about 1079 concepts, while only 1038 are numbered consecutively in the database, and the rest are numbered alphabetically, and the entry 929 is missing. Due to a lack of further information, we decided to limit the number of concepts in the list to those 1037 items that are given direct numbers, but we cannot tell if this reflects the numbering of the original.

Reply:
Thanks!